blocks/large-promo-block/features/large-promo/default.jsx

```diff
@@ -282,6 +283,7 @@ const LargePromoItem = ({ customFields, arcSite }) => {
 	}
 
 	const { editableContent, searchableField } = useEditableContent();
+	const editable = true;
```

Comment:
This will always be true and is not necessary.

Reply:
@nschubach removed editable